dev/com.ibm.ws.security.openidconnect.client/resources/com/ibm/ws/security/openidconnect/client/internal/resources/OidcClientMessages.nlsprops

```diff
@@ -34,10 +34,11 @@ OIDC_CLIENT_CONFIG_MODIFIED.useraction=None.
 #1702 is used by clients.common bundle, do not use.
 
 # used_by_client_project and clients project IF YOU CHANGE THIS YOU MUST DUPLICATE THE CHANGE IN THE OTHER BUNDLE!
+# social also uses this and it does not have the enforceHTTPS attribute.
 #do not translate enforceHTTPS
-OIDC_CLIENT_URL_PROTOCOL_NOT_HTTPS=CWWKS1703E: The OpenID Connect client requires SSL (HTTPS) but the OpenID Connect provider URL is HTTP: [{0}].  Update the configuration so that [enforceHTTPS] attribute matches the target URL scheme. 
+OIDC_CLIENT_URL_PROTOCOL_NOT_HTTPS=CWWKS1703E: The OpenID Connect client requires SSL (HTTPS) but the OpenID Connect provider URL is HTTP: [{0}].  Update the configuration to use an HTTPS URL, or use the OpenID Connect client feature and set [enforceHTTPS] to false. 
```

Comment:
- Update so there's only one space between the two sentences.
- Change "OpenID Connect client feature" to "OpenID Connect Client feature" (since it's the proper name of the feature)

Reply:
Done, I said "use the blah feature" instead of "enable the blah feature" as if they need that attribute then they need to use the oidc feature and NOT use the social login feature.